platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -225,71 +233,71 @@ protected void instanceSingleton(SingletonBase s) {
 	public FrameLayout layout;
 	public RelativeLayout adLayout;
 
-
 	static public GodotIO io;
 
 	public static void setWindowTitle(String title) {
 		//setTitle(title);
 	}
 
-
 	static SingletonBase singletons[] = new SingletonBase[MAX_SINGLETONS];
-	static int singleton_count=0;
-
+	static int singleton_count = 0;
 
 	public interface ResultCallback {
 		public void callback(int requestCode, int resultCode, Intent data);
 	};
+
 	public ResultCallback result_callback;
 
 	private PaymentsManager mPaymentsManager = null;
 
-	@Override protected void onActivityResult (int requestCode, int resultCode, Intent data) {
-		if(requestCode == PaymentsManager.REQUEST_CODE_FOR_PURCHASE){
+	@Override
+	protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+		if (requestCode == PaymentsManager.REQUEST_CODE_FOR_PURCHASE) {
 			mPaymentsManager.processPurchaseResponse(resultCode, data);
-		}else if (result_callback != null) {
+		} else if (result_callback != null) {
 			result_callback.callback(requestCode, resultCode, data);
 			result_callback = null;
-		};
+		}
+		;
```

Comment:
Another stray `;`.

Reply:
Do you want me to remove it the auto-lint commit instead of the manual one ?